--- origsrc/hcal-gtk-0.6.3/Makefile.am	2006-04-20 04:11:24.000000000 -0500
+++ src/hcal-gtk-0.6.3/Makefile.am	2006-12-16 23:19:59.536750000 -0600
@@ -1,4 +1,4 @@
-SUBDIRS = intl src po
+SUBDIRS = src po
 
 install-data-local:
 	if test -d $(srcdir)/data; then \
--- origsrc/hcal-gtk-0.6.3/configure.in	2006-04-24 09:06:06.000000000 -0500
+++ src/hcal-gtk-0.6.3/configure.in	2006-12-16 23:44:36.561750000 -0600
@@ -57,9 +57,10 @@
 dnl *************************************************
 dnl gettext support
 dnl *************************************************
+GETTEXT_PACKAGE="hcal-gtk"
+AC_SUBST(GETTEXT_PACKAGE)
 ALL_LINGUAS="he"
-AM_GNU_GETTEXT
-AM_GNU_GETTEXT_VERSION(0.10.40)
+AM_GLIB_GNU_GETTEXT
 
 if test "x${prefix}" = "xNONE"; then
 AC_DEFINE_UNQUOTED(LOCALE_DIR, "${ac_default_prefix}/${DATADIRNAME}/locale", [Name of gettext locale directory])
--- origsrc/hcal-gtk-0.6.3/src/Makefile.am	2006-04-24 07:11:02.000000000 -0500
+++ src/hcal-gtk-0.6.3/src/Makefile.am	2006-12-16 23:20:31.739875000 -0600
@@ -38,8 +38,8 @@
 
 INCLUDES = \
 	-DPACKAGE=\"$(PACKAGE)\" \
-	-DPIXMAP_DIR=\"$(DESTDIR)$(datadir)/pixmaps/$(PACKAGE)\" \
-	-DPACKAGE_DATA_DIR=\"$(DESRDIR)$(datadir)/$(PACKAGE)\" \
+	-DPIXMAP_DIR=\"$(datadir)/pixmaps/$(PACKAGE)\" \
+	-DPACKAGE_DATA_DIR=\"$(datadir)/$(PACKAGE)\" \
 	@GTK_CFLAGS@
 
 hcal_gtk_LDFLAGS =
